### PR TITLE
Added GPT2-LM-HEAD model

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/adapters/nlp.py [hunk 1]
```diff
@@ -121,7 +121,10 @@ def configure(self):
     def process(self, raw, identifiers=None, frame_meta=None):
         raw_output = self._extract_predictions(raw, frame_meta)
         result = []
-        for identifier, token_output in zip(identifiers, raw_output[self.logits_out]):
+        logits = raw_output[self.logits_out]
+        if len(logits.shape) == 4:
+            logits = logits[0]
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/nlp.py:126]: could you please provide example which shape expected and how it works?

Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/adapters/nlp.py:126]: As a result, we need to get the last 2 dimensions in `token_output`. `logits` shape can be either 1, 1024, 50257 or 1, 1, 1024, 50257. To handle the second case, added this code

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/nlp.py:126]: np.squeeze(logits, axis=1), here make sence to remove second dim instead of first which contains batch

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/nlp.py:126]: and it is also can be done inside cycle, not before (e.g. if model will be run with batch size > 1), we will have output [42, 1, 1024, 50257]
